src/services/boardService.js

```diff
@@ -0,0 +1,14 @@
+
+export function makeBoard(ranks, files) {
+  return ranks.reduce((board, rank) => {
+    files.forEach(file => {
+      const id = `${file}${rank}`;
+      board.push({ id: id, piece: null});
+    });
+    return board;
+  }, []);
+}
+
+export const FILES = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
+export const RANKS = [1, 2, 3, 4, 5, 6, 7, 8];
+export const BOARD = makeBoard(RANKS, FILES);
```

Comment:
I was thinking the FILES and RANKS can just be used in the function. Maybe it's better to leave it like this to make it more flexible and testable. This was a good thing - bringing those two in here - nice!